web-console/src/pages/_app.tsx

```diff
@@ -59,9 +58,7 @@ if (themeConfig.routingLoader) {
 // provide the default query function to your app with defaultOptions
 const queryClient = new QueryClient({
   defaultOptions: {
-    queries: {
-      queryFn: defaultQueryFn
-    }
+    queries: {}
```

Comment:
do we still need `defaultOption` here if queries is empty? do we need this at all anymore?